scripts/mutect2_wdl/mutect2_multi_sample.wdl

```diff
@@ -152,5 +154,7 @@ workflow Mutect2_Multi {
         Array[File] contamination_tables = Mutect2.contamination_table
 
         Array[File?] oncotated_m2_mafs = Mutect2.oncotated_m2_maf
+        Array[File?] m2_merged_bam_out = Mutect2.merged_bam_out
```

Comment:
Likewise about naming the variable without "merged"

Reply:
Done